module/zfs/dataset_kstats.c

```diff
@@ -37,6 +37,8 @@ static dataset_kstat_values_t empty_dataset_kstats = {
 	{ "nread",	KSTAT_DATA_UINT64 },
 	{ "nunlinks",	KSTAT_DATA_UINT64 },
 	{ "nunlinked",	KSTAT_DATA_UINT64 },
+	{ "prunes",	KSTAT_DATA_UINT64 },
+	{ "prunes_delayed",	KSTAT_DATA_UINT64 },
```

Comment:
Would `pruned_skipped` be more accurate since it's possible we might skip pruning this superblock multiple times for the same delay?

Reply:
I originally used `skipped` but thought `delayed` relates it to the `zfs_arc_prune_delay_ms` parameter. Technically `skipped` is more accurate. Maybe `prunes_skipped`, and change the parameter to `zfs_arc_prune_skip_ms`, with the explanation "skip pruning this superblock for some milliseconds..."?

Reply:
Yeah, I like that.